cli/src/commands/git.rs

```diff
@@ -457,6 +457,12 @@ fn cmd_git_clone(
         ));
     }
 
+    let colocate = if command.settings().git_settings().colocate {
```

Comment:
I think that if we have this option, then the commands should have a `--no-colocate` command-line flag.

Reply:
(I'm not sure if that's a great name, though)